Kernel/Bus/USB/Drivers/HID/Codes.h

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2023, Liav A. <liavalb@hotmail.co.il>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/StdLibExtraDetails.h>
+#include <AK/StringView.h>
+#include <AK/Types.h>
+
+namespace Kernel::USB::HID {
+
+enum class SubclassCode : u8 {
+    BootProtocol = 0x01,
+};
```

Comment:
It would be nice if there was some spec link added somewhere in this code.

Reply:
I don't think this is crucial here. I could try to find the reference from a USB specification if it's really necessary.